docx_to_markdown.py

```
@@ -0,0 +1,50 @@
+import docx
+import re
```

Ruff (0.11.9)

`re` imported but unused

Remove unused import: `re`

(F401)